src/main/java/core/basesyntax/service/impl/OperationParserImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationParser;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OperationParserImpl implements OperationParser {
+    private static final int HEADER = 0;
+    private static final String SPLITTER = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseOperation(List<String> data) {
+        data.remove(HEADER);
```

Comment:
you don't need to remove HEADER. You will skip it in stream and retutn list without header.
```suggestion

```